content/resources/user/submit-data/sower.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Gen3 - Data Contribution"
+date: 2018-09-12T20:53:20-05:00
+linktitle: /resources/user/submit-data
+layout: withtoc
+menuname: userMenu
+---
+{{% markdownwrapper %}}
+# Linking Data from external Data Clouds to Gen3 Data Commons
+***
+
+Data files that are stored on other cloud services, such as Amazon Web Services (AWS) or Google Cloud Storage (GCS), can be made available in a Gen3 data commons without moving it. There are several technical details for individual cases, and thus, you will need to get in contact with us for further inquiries. Below we list some guidelines that should prepare for specific processes such as indexing and submission.
+
+1. Evaluate the bucket structure, consent groups and authorization to individual projects. Consent groups and authorization are *key* parameters that need to be communicated with us and data/bucket owners. Moreover, a flat bucket structure is preferred, but nested structure is also acceptable if needed.
```

Comment:
same here, maybe don't say "communicated with us" and instead say, "communicated with the data commons operator" or just "communicated"?